config/samples/apps_v1alpha1_nemocustomizer.yaml

```diff
@@ -9,15 +9,16 @@ metadata:
 spec:
   wandbSecret:
     name: wandb-secret
-    apiKeyKey: encryption_key
+    apiKeyKey: encryptionKey
```

Comment:
I might not have the most update to date doc, but the NMP helm template is still defaulting the wandb api key in secret as `encryption_key`
https://gitlab-master.nvidia.com/aire/microservices/nmp/-/blob/main/helm/platform/components/customizer/templates/_helpers.tpl#L144

Reply:
This is to align with the default `key` that we have used in the ansible playbook [here](https://github.com/NVIDIA/k8s-nim-operator/blob/main/test/e2e/nemo-dependencies/customizer/templates/secrets.yaml.j2#L19)